Makefile

```diff
@@ -228,7 +228,7 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(PLATFORM)-$(
 	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(PLATFORM)-$(TARGET_ARCH)-app \
 	| $(BUILD_DIRECTORY)
 	mkdir $@
-	$(NPX) pkg --output $@/$(ETCHER_CLI_BINARY) -t node6-$(PLATFORM_PKG)-$(TARGET_ARCH) $</$(ENTRY_POINT_CLI)
+	cd $< && ../../$(NPX) pkg --output ../../$@/$(ETCHER_CLI_BINARY) -t node6-$(PLATFORM_PKG)-$(TARGET_ARCH) $(ENTRY_POINT_CLI)
```

Comment:
This looks a little bit ugly - I wonder if it'd be less ugly to have a pkg-wrapper script, that could 'hide' this ugliness from the `Makefile`?
Would it be also worth adding an issue to the `pkg` repo, suggesting they add an additional command-line parameter, that would then make pkg 'natively' do what you want (without this wrapper script)?

Reply:
See https://github.com/zeit/pkg/issues/231. I'll merge this so we can release (we're planning a release very soon), but hopefully `pkg` fixes the issue soon, and we can update this.